src/app/pages/messages/messages.component.html

```diff
@@ -1,3 +1,3 @@
-<p>
-  messages works!
-</p>
+<div class="page">
```

Comment:
Don't you need to use *ngFor to iterate through all the conversations?

Reply:
Yes, definitely will.  Just wanted to get this initial boiler plate into dev.. I plan on polishing it up in future pull requests